packages/zero-runtime/src/index.ts

```diff
@@ -1,6 +1,9 @@
 import styled from './styled';
-import sx from './sx';
-import keyframes from './keyframes';
 
-export { styled, sx, keyframes };
+export { default as sx } from './sx';
+export { default as keyframes } from './keyframes';
+
+export type { ThemeArgs } from './base';
+
+export { styled };
 export default styled;
```

Comment:
We didn't have styled exported as a default export before, I would keep it that way, especially considering that this package is more than just the styled() util.